independent-publisher-connectors/AfterShip/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2690 @@
+                        },
+                        "tracking_origin_country": {
+                          "type": "string",
+                          "description": "tracking_origin_country"
+                        },
+                        "tracking_destination_country": {
+                          "type": "string",
+                          "description": "tracking_destination_country"
+                        },
+                        "tracking_key": {
+                          "type": "string",
+                          "description": "tracking_key"
+                        },
+                        "tracking_postal_code": {
+                          "type": "string",
+                          "description": "tracking_postal_code"
+                        },
+                        "tracking_ship_date": {
+                          "type": "string",
+                          "description": "tracking_ship_date"
+                        },
+                        "tracking_state": {
+                          "type": "string",
+                          "description": "tracking_state"
+                        },
+                        "active": {
+                          "type": "boolean",
+                          "description": "active"
+                        }
+                      },
+                      "description": "tracking"
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/trackings/{slug}/{tracking_number}/mark-as-completed": {
+      "post": {
+        "summary": "Mark tracking as completed",
+        "description": "Mark a tracking as completed. The tracking won't auto update until retrack it.",
+        "operationId": "MarkTrackingAsCompleted",
+        "parameters": [
+          {
+            "name": "slug",
+            "default": "hong-kong-post",
+            "in": "path",
+            "type": "string",
+            "required": true,
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "tracking_number",
+            "default": "RA123456789HK",
+            "in": "path",
+            "type": "string",
+            "required": true,
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "description": "Content-Type",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "reason": {
+                  "type": "string",
+                  "description": "reason",
+                  "title": "",
```

Comment:
@taiki-yoshida Empty property values can be removed until unless you have a specific reason to keep them.